pkg/rules/interfaces.go

```diff
@@ -0,0 +1,13 @@
+package rules
+
+import (
+	"context"
+)
+
+type ServiceInterface interface {
+	ListRules(context.Context, int64, int64) ([]*RuleData, error)
+	GetRule(context.Context, string) (*RuleData, error)
+	UpdateRule(context.Context, string, RuleData) error
+	CreateRule(context.Context, string, RuleData, string) error
```

Comment:
apart from reordering the args we need to drop the filename location

Reply:
Interface redone. No more filename args.